test/otel/testdata/collector-config.yaml

```diff
@@ -15,6 +15,9 @@ exporters:
 extensions:
   ddflare:
     include_metadata: true  # Set to true for testing; adjust for production.
+  # ddprofiling:
```

Comment:
Is there a reason to have this code commented?

Reply:
This will break OCB until: https://github.com/open-telemetry/opentelemetry-collector/issues/12162 is fixed. The PR is up, but not merged yet.